### PR TITLE
ISOM-1154: implement simple layout for edit page

### PR DESCRIPTION
**Pull request by harishv7**
## Problem

We need a simple left-right layout for editing page

Closes ISOM-1154

## Solution

Implements a basic layout and a grid division for editor(sidebar) and preview

**Breaking Changes**

- [ ] Yes - this PR contains breaking changes
  - Details ...
- [X] No - this PR is backwards compatible

### CODE REVIEW

File: apps/studio/src/pages/dashboard/edit/index.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+const EditPage: NextPageWithLayout = () => {
+  return (
+    <Grid w="100vw" templateColumns="repeat(3, 1fr)" gap={4}>
+      {/* TODO: Implement sidebar editor */}
+      <GridItem w="100%" colSpan={1} bg="tomato">
+        <h1>Editor sidebar appears here</h1>
+      </GridItem>
+      {/* TODO: Implement preview */}
+      <GridItem w="100%" colSpan={2} bg="papayawhip">
```
Comment (kishore03109) [inline apps/studio/src/pages/dashboard/edit/index.tsx:13]: no tokens :(

File: apps/studio/src/templates/layouts/PageEditingLayout.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+import { AppNavbar } from '~/components/AppNavbar'
+import { EnforceLoginStatePageWrapper } from '~/components/AuthWrappers'
+import { DashSidebar } from '~/components/DashSidebar'
+import { APP_GRID_TEMPLATE_AREA } from '~/constants/layouts'
+import { type GetLayout } from '~/lib/types'
+
+export const PageEditingLayout: GetLayout = (page) => {
+  return (
+    <EnforceLoginStatePageWrapper>
+      <Flex minH="$100vh" flexDir="column" bg="base.canvas.alt" pos="relative">
```
Comment (kishore03109) [inline apps/studio/src/templates/layouts/PageEditingLayout.tsx:11]:
```suggestion
      <Flex minH="100vh" flexDir="column" bg="base.canvas.alt" pos="relative">
```

**Review by kishore03109 — APPROVED** [review]
fix the typo can alr

File: apps/studio/src/pages/dashboard/edit/index.tsx (hunk 1 above)
Comment (harishv7) [reply apps/studio/src/pages/dashboard/edit/index.tsx:13]: yes just placeholders for now, will remove and replace in implementation ticket

**Pull request merged**